index.html

```diff
@@ -2653,7 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
-          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li><a>Trim</a>(<var>value</var>) and convert <a>to ASCII lowercase</a></li>
```

Comment:
OK I see that we use to ASCII lowercase elsewhere and that is from https://html.spec.whatwg.org/#converted-to-ascii-lowercase

so that might be better... I wonder if there is something for trim as well